numpy/lib/index_tricks.py

```diff
@@ -68,6 +71,15 @@ def ix_(*args):
     array([[2, 4],
            [7, 9]])
 
+    >>> ixgrid = np.ix_([True,True], [2,4])
+    >>> a[ixgrid]
+    array([[2, 4],
+           [7, 9]])
+    >>> ixgrid = np.ix_([True,True], [False, False, True, False, True])
```

Comment:
Spaces after ',' is preferred. Note that the examples date to 2009, before we started getting stricter about style. That, and folks tend to cut corners in interactive code.